cmd/configure/survey.go

```diff
@@ -137,6 +139,22 @@ func (c *CmdConfigure) askValue(q question) string {
 		return c.askBoolValue(label)
 	}
 
+	if q.valueType == templates.ParamValueTypeChargeModes {
+		chargingModes := []string{string(api.ModeOff), string(api.ModeNow), string(api.ModeMinPV), string(api.ModePV)}
+		chargeModes := []string{
+			c.localizedString("ChargeModeOff", nil),
```

Comment:
macht die Übersetzung Sinn wenn ich im Gui was Anderes sehe? Oder ist das auch schon mehrsprachig?

Reply:
Das relevante da sehe ich eher im Erklärtext zu den Modi. Schaus dir einfach mal an. Übersetzt ist da auch nur "Aus" und "Sofort". Und das kann man im englischen lassen und dann in die Erklärung reinbringen.

Reply:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/42185/145707821-bfb98dac-b360-4c10-85f8-695c13a659bd.png">

Das sehe ich in der UI. Müsste also noch aus "Aus" dann "Stop" machen.